lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
Okay I gothcha now. So there are some places where you are doing a `.then(....` and others where you are doing a `asCallback(....`. Why the difference?